monitoring/README.md

```diff
@@ -11,7 +11,10 @@ This strategy uses 3 components:
 
 ## Prerequisites
 
-- Install [Docker](https://www.docker.com/)
+```
+$ brew install colima docker docker-compose
```

Comment:
I thought docker-compose was included in docker now? Is it just true for docker desktop?

Reply:
Yep, `brew install docker` only install the `docker` binary.